pkg/proxy/userspace/proxysocket.go

```diff
@@ -188,19 +188,19 @@ func (udp *udpProxySocket) Addr() net.Addr {
 }
 
 // Holds all the known UDP clients that have not timed out.
-type clientCache struct {
-	mu      sync.Mutex
-	clients map[string]net.Conn // addr string -> connection
+type ClientCache struct {
+	sync.Mutex
```

Comment:
ack, changed. 

From a curiosity perspective, is there a particular reason?  `ClientCache.Lock()` looks nicer to me than `ClientCache.Mu.Lock()`, and IMO more clearly conveys the direct intent of the action (but that's really just a matter of opinion).